src/backend/marsha/core/utils/medialive_utils.py

```diff
@@ -304,6 +317,17 @@ def create_mediapackage_harvest_job(video):
     )
 
 
+def delete_mediapackage_channel(video):
+    """Delete a mediapackage channel."""
+    channel_id = video.live_info.get("mediapackage").get("channel").get("id")
```

Comment:
As we got many identical chained get, maybe we could add some small helper functions, like
```python
def get_mediapackage_channel_id(video):
    return video.live_info.get("mediapackage").get("channel").get("id")
```
It may simplify other function and abstract video data structure in case of modification.

Reply:
this should be on the `Video` model object ?

Reply:
I think so, yes

Reply:
Done